src/stactools/cli/commands/validate.py

```diff
@@ -1,97 +1,62 @@
+import json
 import sys
-from typing import List, Optional
+from typing import Optional
 
 import click
-import pystac
-from pystac import Catalog, Collection, Item, STACObject, STACValidationError
-
-from stactools.core.utils import href_exists
+from stac_validator.validate import StacValidate
 
 
 def create_validate_command(cli: click.Group) -> click.Command:
     @cli.command("validate", short_help="Validate a stac object.")
     @click.argument("href")
     @click.option(
-        "--recurse/--no-recurse",
+        "--recursive/--no-recursive",
+        help="Recursively validate all stac objects in this catalog.",
         default=True,
-        help=(
-            "If false, do not validate any children "
-            "(only useful for Catalogs and Collections"
-        ),
     )
+    @click.option("--links/--no-links", help="Validate links.", default=True)
+    @click.option("--assets/--no-assets", help="Validate assets.", default=True)
+    @click.option("-v", "--verbose", is_flag=True, help="Enables verbose output.")
     @click.option(
-        "--links/--no-links",
-        default=True,
-        help=("If false, do not check any of the objects's links."),
+        "--quiet/--no-quiet", help="Do not print output to console.", default=False
```

Comment:
I think the help needs to always be the positive form, rather than the default behavior?

```suggestion
        "--quiet/--no-quiet", help="Print output to console.", default=False
```

Reply:
Does it? This is the current output:

```text
$ stac validate --help
Usage: stac validate [OPTIONS] HREF

  Validates a STAC object.

  This is a thin wrapper around [stac-validate](https://github.com/stac-
  utils/stac-validator). Not all command-line options are exposed. If you want
  more control over validation, use `stac-validator` directly.

  If you'd like linting, use `stac lint`.

Options:
  --recursive / --no-recursive  Recursively validate all STAC objects in this
                                catalog.
  --links / --no-links          Validate links.
  --assets / --no-assets        Validate assets.
  -v, --verbose                 Enables verbose output.
  --quiet / --no-quiet          Do not print output to console.
  --log_file TEXT               Save full recursive output to log file (local
                                filepath).
  --help                        Show this message and exit.
```

I think it'd be better maybe just to eliminated the `--no-quiet` option, which should make the help clearer. What do you think?

Reply:
oh, right, quiet actually makes it do that. I think it's right as it is. Though, does quiet also not print it to the log file? If so maybe remove "to console"

Reply:
> does quiet also not print it to the log file?

No, the log file is internal to `stac-validator` so doesn't see the `--quiet` argument at all.